src/popup/router/index.js

```diff
@@ -34,6 +34,10 @@ const router = new VueRouter({
   scrollBehavior: (to, from, savedPosition) => savedPosition || { x: 0, y: 0 },
 });
 
+if (window.IS_DESKTOP) {
+  router.afterEach(() => document.querySelector('#app').scroll(0, 0));
```

Comment:
I made this fix because of https://github.com/vuejs/vue-router/blob/7fefb813a0f4b3f63cff7361eeed2bf8a82f8f51/dist/vue-router.js#L1948
Looks like vue is always scrolling `window`, but on desktop we need to scroll `#app`.